### DISCUSSION TITLE
Can we use this in bitbucket?

### DISCUSSION THREAD

**Discussion by anurawat01** [intuit/auto#2095] [Q&A] [answered]
Can we integrate this thing in bitbucket repo as it does not contain label feature while creating a PR so this might be the issue?
Any developer waiting for your kind response?

**Comment by Sourabh-Munat** [comment]
I would also like to know if this can be used with bitbucket or not ?

**Comment by hipstersmoothie** [comment, marked as answer]
If bitbucket doesn't support labels it's gonna break a lot of assumptions with auto. That said I would like to support other platforms someday #203. But it a lot of work and I don't see myself doing it anytime soon.
